pkg/k8s/apis/cilium.io/client/crds/v2/ciliumendpoints.yaml

```diff
@@ -480,6 +480,21 @@ spec:
                 - disconnected
                 - invalid
                 type: string
+              workload:
+                description: Workload is the workload information of the pod associated
+                  with this endpoint
+                properties:
+                  kind:
+                    description: Kind of the workload (Deployment, Statefulset, Daemonset,
+                      etc.)
+                    type: string
+                  name:
+                    description: Name of the workload
+                    type: string
+                  namespace:
+                    description: Namespace of the workload
+                    type: string
```

Comment:
Nit. Is a namespace field really required here? I'd expect it to be always implied by the namespace where the pod (hence the CiliumEndpoint) resides.

Reply:
I think I agree, in this context the Workload is always accessed through Endpoint or CiliumEndpoint and therefore we can always infer the namespace.

Reply:
Cross namespace owner refs are invalid, so I think this is correct.

Reply:
I agree that namespace should not be necessary in the CEP changes. However in the hubble APIs I think it could be helpful to include the namespace there since at least from the example in the PR description the namespace is not known for that traffic.

Reply:
Ha thats on me for filtering out the flow with a jq query. We should have the namespace set on source/destination endpoint structs in Flow.